specification/metadata/schema/precision.md

```diff
@@ -0,0 +1,17 @@
+# Precision
```

Comment:
Maybe Number Precision, to be clear and explicit?

Reply:
"Numeric Precision"?